crates/common/tedge_utils/src/fs_notify.rs

```diff
@@ -91,11 +91,11 @@ impl WatchDescriptor {
     #[cfg(test)]
     #[cfg(feature = "fs-notify")]
     /// get a set of `Masks` for a given `dir_path`
-    pub(super) fn get_mask_set_for_directory(&mut self, wid: c_int) -> Vec<FileEvent> {
+    pub(super) fn get_mask_set_for_directory(&mut self, wid: c_int) -> HashSet<FileEvent> {
```

Comment:
`get_mask_set_for_directory` the name and what the function is doing does not seem to be right

Reply:
That's what exactly it does. It just collects all the `event masks` to that specific `directory`.

Reply:
It's more "for descriptor id" or "for root/parent directory"

Reply:
because I would expect to give a directory as input to something called get_mask_set_for_directory, but its actually taking an id as input

Reply:
Yes, it's for the `wid`, I missed that part, I will rename that.